src/app/src/utils.js

```diff
@@ -131,6 +131,7 @@ export function getBoundaryPermissions({ boundary, user }) {
         canWrite: false,
```

Comment:
I'll be updating this function again in the PR for #151 to separate the role and boundary aspects of these permissions. It might not be worth getting into too much detail here.